templates/part.board.mainView.php

```diff
@@ -66,8 +69,18 @@
 						<p><?php p($l->t('Drop your files here to upload it to the card')); ?></p>
 					</div>
 					<div data-as-sortable-item-handle>
-						<div class="card-upper">
-							<h4>{{ cardservice.get(c.id).title }}</h4>
+						<div class="card-upper">						 
+							<h4>
+								<span class="editable-inline"
+									ng-show="!c.status.editCard"
+									ng-click="c.status.editCard=true">{{cardservice.get(c.id).title}}</span>
+
+								<form ng-if="c.status.editCard" ng-submit="cardservice.update(c); c.status.editCard=false">
+									<input type="text" placeholder="<?php p($l->t('Add a new card')); ?>"
```

Comment:
Can you add the input-inline class here, this should make the input look like the other inline editing fields. Also we need some adjustment so that the height and text of the card is not flipping around when changing from view to edit mode.

Reply:
Added the class, struggling with controlling the height and text. Not sure where the extra height is coming from. Any advice on how to track it down? css issues like this is usually a struggle for me.